### PR TITLE
fix: price format

### LINKED ISSUE #665: [FEATURE] Display correct pricing
### Description

Use info from #263 

### Acceptance criteria

- [x] implement pricing logic in helpers
- [x] add pricing table in product page
- [x] add pricing explanation in docs

### PR DESCRIPTION
**Pull request by mkucmus**
closes #665 

## Changes
- fixes displaying the prices. allows the price to be undefined (do not force it to be 0.00 otherwise)
- displays tier prices table

### Checklist

- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: packages/helpers/src/product/getProductTierPrices.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Product } from "@shopware-pwa/commons/interfaces/models/content/product/Product";
+
+/**
+ * @beta
+ */
+interface TierPrice {
```
Comment (patzick) [inline packages/helpers/src/product/getProductTierPrices.ts:6]:
```suggestion
export interface TierPrice {
```

File: packages/helpers/src/product/getProductTierPrices.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Get the prices depending on quantity added to cart.
+ * Tier prices can be set in `Advanced pricing` tab in `Product view` (admin panel)
+ *
+ * @returns TierPrice[]
+ * @beta
+ */
+export function getProductTierPrices(
+  product: Product
+): TierPrice[] | undefined {
```
Comment (patzick) [inline packages/helpers/src/product/getProductTierPrices.ts:21]: can we always return an empty array as a fallback?
```suggestion
): TierPrice[] {
```

Comment (mkucmus) [reply packages/helpers/src/product/getProductTierPrices.ts]: :+1: you're right. fixed

Comment (mkucmus) [reply packages/helpers/src/product/getProductTierPrices.ts]: done

**Pull request merged**